src/components/formattingUtils/Formatting.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react'
+
+interface TitleDateProps {
+  title: string
+  date: string
+}
+
+export const FormattedTitleDate = ({ title, date }: TitleDateProps): ReactElement => {
+  return (
+    <div className="flex flex-col md:flex-row md:justify-between">
+      <span className="truncate">{title}</span>
```

Comment:
hast du das auf mobile gecheckt ? Weil ich glaube truncat geht nur wenn der Text in einer Zeile ist.

Reply:
yes das klappt